### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by joredudiaz**
## Summary

- footer element
- footer style

## Jira Story

https://trello.com/c/M7HxqOFn

## Screenshot

image

### CODE REVIEW

File: mock-up/index.html [hunk 1]
```diff
@@ -82,9 +82,15 @@ <h1 class="title-words">Back-End Tools</h1>
       <img src="assets/django_icon.svg" alt="django" class="tech-img">
       <p class="tech-name">Django</p>
       <p class="tech-release">Release: 2005</p>
-      <p class="tech-description">Django (/ˈdʒæŋɡoʊ/ JANG-goh; stylised as django)[4] is a Python-based free and open-source web framework that follows the model-template-view (MTV) architectural pattern.</p>
+      <p class="tech-description">Django (/ˈdʒæŋɡoʊ/ JANG-goh; stylised as django) is a Python-based free and open-source web framework that follows the model-template-view (MTV) architectural pattern.</p>
     </div>
   </div>
+
```
Comment (FrankIglesias) [inline mock-up/index.html]: remove extraline

File: mock-up/index.html [hunk 2]
```diff
@@ -82,9 +82,15 @@ <h1 class="title-words">Back-End Tools</h1>
...
+  <div class="footer">
```
Comment (FrankIglesias) [inline mock-up/index.html]: use footer tag

File: mock-up/scss/footer.scss [hunk 3]
```diff
@@ -0,0 +1,20 @@
+.footer {
```
Comment (FrankIglesias) [inline mock-up/scss/footer.scss:1]: Check indentation

File: mock-up/scss/tech.scss [hunk 4]
```diff
@@ -44,8 +44,8 @@
 
 .tech-img {
   height: 100px;
-  object-fit: cover;
-  width: 120px;
+  object-fit: contain;
```
Comment (FrankIglesias) [inline mock-up/scss/tech.scss]: cover instead

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: mock-up/scss/colors.scss [hunk 5]
```diff
@@ -1,3 +1,4 @@
 $earls-green: #BED23C;
 $white: white;
 $gray: gray;
```
Comment (betoobernal) [inline mock-up/scss/colors.scss]: use hexadecimal.

File: mock-up/index.html [hunk 6]
```diff
@@ -59,7 +59,7 @@ <h1 class="title-words">Front-End Tools</h1>
     <h1 class="title-words">Back-End Tools</h1>
   </div>
 
-  <div class="techs">
+  <div class="techs-container">
     <div class="tech">
       <img src="assets/rails_icon.png" alt="rails" class="tech-img">
```
Comment (betoobernal) [inline mock-up/index.html]: sorry. I forget to tell you that you must close img tag. ``
This allows us to treat it as if it were well-formed XML. Please fixes all

File: mock-up/index.html [hunk 7]
```diff
@@ -82,9 +82,15 @@ <h1 class="title-words">Back-End Tools</h1>
...
       <p class="tech-name">Django</p>
       <p class="tech-release">Release: 2005</p>
-      <p class="tech-description">Django (/ˈdʒæŋɡoʊ/ JANG-goh; stylised as django)[4] is a Python-based free and open-source web framework that follows the model-template-view (MTV) architectural pattern.</p>
+      <p class="tech-description">Django (/ˈdʒæŋɡoʊ/ JANG-goh; stylised as django) is a Python-based free and open-source web framework that follows the model-template-view (MTV) architectural pattern.</p>
     </div>
   </div>
+
+  <div class="footer">
+    <h3 class="footer-title">WE ARE GAME CHANGERS</h3>
+    <img src="assets/wolox_logo_white.svg" alt="wolox" class="footer-logo">
```
Comment (betoobernal) [inline mock-up/index.html]: same

File: mock-up/index.html [hunk 8]
```diff
@@ -82,9 +82,15 @@ <h1 class="title-words">Back-End Tools</h1>
       <img src="assets/django_icon.svg" alt="django" class="tech-img">
```
Comment (betoobernal) [inline mock-up/index.html]: same

File: mock-up/scss/footer.scss [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+.footer-title {
+  color: $white;
+  font-size: 26px;
+  margin-bottom: 20px;
+}
+
+.footer-copyright {
+  color: $white;
+  margin-top: 20px;
+  margin-right: 20px;
```
Comment (betoobernal) [inline mock-up/scss/footer.scss]:
```suggestion
  margin: 20px 20px 0 0;
```

**Review by betoobernal — APPROVED** [review]

**Pull request merged**